.github/workflows/awstest.yml

```diff
@@ -25,13 +25,16 @@ jobs:
         run: conda install -c conda-forge awscli
       - name: Start AWS batch job
         env:
-          AWS_ACCESS_KEY_ID: ${{secrets.AWS_KEY_ID}}
-          AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_KEY_SECRET}}
-          TOWER_ACCESS_TOKEN: ${{secrets.TOWER_ACCESS_TOKEN}}
+          AWS_ACCESS_KEY_ID: ${{secrets.AWSTEST_KEY_ID}}
+          AWS_SECRET_ACCESS_KEY: ${{secrets.AWSTEST_KEY_SECRET}}
+          TOWER_ACCESS_TOKEN: ${{secrets.AWSTEST_TOWER_TOKEN}}
+          AWS_JOB_DEFINITION: ${{secrets.AWS_JOB_DEFINITION}}
+          AWS_JOB_QUEUE: ${{secrets.AWS_JOB_QUEUE}}
+          AWS_S3_BUCKET: ${{secrets.AWS_S3_BUCKET}}
```

Comment:
Any reason why some secrets start with `AWS` and some `AWSTEST`? 😅  It's also a fraction confusing that the env names don't match the secret names. But I guess that this is more of an upstream comment sorry, just noticed it here...

Reply:
My logic was to change the name of the first three secrets, as these are the ENV variables that a user could have set privately to the own tower token and AWS credentials. But I can see that this is confusing, to have some things with `AWS` and some with `AWSTEST`

I'll fix it in the template before too many pipelines adopt it, sorry for the confusion!

Reply:
Ah ok, that makes sense 👍 Yeah I see your logic, but I wonder if we are less likely to mess this up if we just use `AWS` instead of `AWSTEST` for everything (also for others who want to run this in their own pipelines from the template, with their own GitHub Actions secrets).

Reply:
If it's not too much trouble to change anyway - if it's already a big hassle then it's fine to leave as it is.

Reply:
not, it's not, I'll fix it, it is a good point :)